bin/docker-release

```diff
@@ -1,2 +1,18 @@
 #!/usr/bin/env ruby
-raise NotImplementedError
+
+THIS_FILE = File.symlink?(__FILE__) ? File.readlink(__FILE__) : __FILE__
```

Comment:
You can actually make this a pathname object in ruby, which will simplify the use you make of it below